libs/model/__tests__/seed/seed.spec.ts

```diff
@@ -0,0 +1,392 @@
+import {
+  BalanceType,
+  ChainBase,
+  ChainNetwork,
+  ChainType,
+  NotificationCategories,
+  dispose,
+} from '@hicommonwealth/core';
+import { expect } from 'chai';
+import { step } from 'mocha-steps';
+import z from 'zod';
+import {
+  AddressSchema,
+  ChainNodeSchema,
+  CommunityContractSchema,
+  CommunitySchema,
+  CommunityStakeSchema,
+  ContractSchema,
+  NotificationCategorySchema,
+  ProfileSchema,
+  SchemaWithModel,
+  SnapshotProposalSchema,
+  SnapshotSpaceSchema,
+  SubscriptionSchema,
+  TopicSchema,
+  UserSchema,
+  checkDb,
+  seed,
+} from '../../src/test';
+
+// testSeed creates an entity using the `seed` function
+// then attempts to find the entity and validate it
+async function testSeed<T extends SchemaWithModel<any>>(
+  schemaModel: T, // zod schema
+  overrides: Partial<z.infer<T['schema']>> = {}, // override attributes
+  options?: {
+    seedOptions?: {
+      // options for seeding
+      noMock: boolean;
```

Comment:
noMock = true or mock = false ? 😆 ... is mock=true the default?

Reply:
I suppose it could be `mock=false` instead of `noMock=true`

Mocking is enabled by default